src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -92,6 +93,14 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeOptions, MapTemplates
         bikeOptions.changeMode(options.selectors);
     }
 
+    function clearDirections() {
+        mapControl.setOriginDestinationMarkers(null, null);
+        mapControl.clearItineraries();
+        itineraryListControl.hide();
+        directionsListControl.hide();
+        $('.directions').removeClass('show-results');
```

Comment:
This should be defined in the selectors object. Same goes for the code not added in this PR that adds the class.